### ISSUE TITLE
Paddy randomly works with Sketch 49.3

### ISSUE THREAD

**Issue by stigmates** [DWilliames/paddy-sketch-plugin#100]
#### Actual behavior:
Nothing happens when I overide/change text label, in a group or in a symbol. But sometimes it works. It's driving me nuts.
I uninsgalled and reboot, tried paddy in different sketch files (new or work in progress doc), no clue of what could it be responsible…

**Comment by blynkkk** [comment]
Same here. Old symbols don't work. When creating new group with "paddied" layer - works

**Comment by stigmates** [comment]
For me it seems not to be a matter of old or new created symbol. This afternoon I duplicated a button previously created and changed the label. It perfectly worked. But half an hour I did exactly the same with the same original symbol to show someone, and it failed! weird…

**Comment by ziyafenn** [comment]
Nothing works

**Comment by Joshuasony** [comment]
Same here. Sometimes it works, most times it doesn't. I have not found out if it is because of my symbols or because of Plugin incompatibility or because of my Sketchapp Version. Would be nice to have more information about those issues. Since Paddy Plugin is released, I'm testing it regularly again and again, without ever having a really functioning version. This plugin has so much potential, it would be really awesome if it worked. 
I also contacted DWilliames on Twitter DM and Mail, but never recieved an answer. I'd totally be ready to help out in this matter, but nobody is reacting on those issues. Maybe here on Github? 
Any updates from the others? Have you tried temporarily disabling all plugins?

**Comment by chsWeb** [comment]
Does anyone have special characters or numbers in any of their file names - either in layer names or parent folders?

**Comment by stigmates** [comment]
Nope

**Comment by Joshuasony** [comment]
@chsWeb, yes, I'm using emojis in page names. Is that the problem?

**Comment by chsWeb** [comment]
@joshuasony I am not sure if the special characters are to blame, but I seem to have this problem most when using special characters any where near a Paddy layer.

**Comment by chsWeb** [comment]
Almost seems like a delay. When it stops working for me, I click off of the Paddy layer, do something else, then come back, click the Paddy layer, then press escape and Paddy kicks in.

**Comment by DWilliames** [comment]
Hey all!! Sorry for the delay – my email has been playing up; and everything automatically went to Spam; so I accidentally missed a lot of emails and updates about these issues.

It seems like the way I initially implemented the 'auto-updating' has become quite flaky with newer Sketch versions. I don't think there's too much I could do to fix it in the current way it is implemented.

In saying that; I began the endeavour of completely re-writing Paddy... and let me tell you; it is a lot faster; and way more reliable so far. (In a recent benchmark test, the newer implementation was no joke, 200x faster!!)

Please bare with me; as I try and reach feature parity with the re-write. I will try and release some sort of Beta of 'Paddy 2' in the next week for you all to test out. Which will hopefully solve all these issues.

Thanks for your patience! 👍

**Comment by johanneslamers** [comment]
@DWilliames 🙌🙏

**Comment by chsWeb** [comment]
Sounds great. JFYI, I think you should post a way for us to send you some donations. I’d gladly contribute to your efforts. Paddy has real potential.

**Comment by johanneslamers** [comment]
@chsWeb you already can make a donation. Check the README. 

If you ***love*** this plugin, why not shout me a coffee ☕️ via [PayPal](https://www.paypal.me/dtw/5) to share the love!
It will definitely help me to be able to support this plugin further.

<a href="https://www.paypal.me/dtw/5">
  <img width="160" height="41" src="https://cdn.rawgit.com/DWilliames/paddy-sketch-plugin/bbb7d6ce/.images/paypal-badge.png" >
</a>

**Comment by DWilliames** [comment]
Hey all! 

Quick update on this: I've released an early preview of the beta now. See #110 or https://github.com/DWilliames/paddy2-beta if you'd like to try it out. 🎉 

Doesn't do everything yet... but I'd be keen to find out if it is more reliable for everyone so far. 🤞

**Comment by marysea** [comment]
Yep – having a similar issue – including when using an icon in a button, when you override the type the whole button shifts to the left and the gap between the icon and the text expands – I've tried all kinds of things and it's just behaving randomly. As others have noted, I quit sketch re-opened and it works for the first override and then goes crazy again. It does the same thing with a button without an icon – again ramdomly.
Looking forward to an update… it's such a life saver! t
